packages/pakistan.provinces/README.md

```diff
@@ -0,0 +1,39 @@
+# 🇵🇰 Paksitan
+
+Contains all the Administrative Territories of Pakistan including four Provinces, two autonomous territories and one federal territory:
+
+Provinces:
+* Balochistan
+* Sindh
+* Punjab
+* Khyber Pakhtunkhwah
+
+Autonomous Territories:
+* Azad Jammu and Kashmir
+* Gilgit Baltistan
+
+Federal Territory:
+* Islamabad 
+
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/pakistan.provinces`
+
+### yarn
+
+`yarn add @svg-maps/pakistan.provinces`
+
+## Credits
+
+The original map is available [here](https://en.wikipedia.org/wiki/File:Pakistan_Districts.svg#filelinks) and free to be used in any custom project.
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Use English names
+* Add viewBox
+* Rename ids
```

Comment:
Could you also indicate that you removed the districts from the original map please?